hooks/post/30-upgrade.rb

```diff
@@ -67,9 +67,9 @@ def remove_gutterball
 end
 
 def remove_event_queue
-  queue_present = `qpid-stat -q --ssl-certificate=/etc/pki/katello/qpid_client_striped.crt -b amqps://localhost:5671 | grep :event | wc -l`.chomp.to_i
+  queue_present = `qpid-stat -q --ssl-certificate=/etc/pki/katello/qpid_client_striped.crt -b amqps://localhost:5671 | grep ':event' | wc -l`.chomp.to_i
```

Comment:
The bug says '' causes the error but that doesn't seem to be true, there's no special characters in `:event` that would be need to be treated as a string literal